src/witan/workspace/components/cassandra.clj

```diff
@@ -48,6 +59,15 @@
 
   component/Lifecycle
   (start [component]
+    (log/info "Bootstrapping Cassandra...")
+    (create-workspace! host keyspace replication-factor)
+    (let [joplin-config (jrepl/load-config (io/resource joplin))]
```

Comment:
OK. That's a big decision on how we will design these components. We can go with it for now.